snakemake/Snakefile

```diff
@@ -139,7 +139,7 @@ if config.get("phylogenetic_tree_newick") == "subjects":
         shell:
             "GToTree -A {input} -H {params.phylogenetic_model} -o phylogeny/gtotree -T IQ-TREE "
                 "-c {params.sequence_length_threshold} -G {params.minimum_hit_fraction} "
-                "-n {threads} -j {threads} > {log} 2>&1 && "
```

Comment:
Minor enhancement to print log statements from this particular program to the screen so the user can monitor progress (can take time to run).